docs/using-gitbase/examples.md

```diff
@@ -140,13 +132,17 @@ Also, you can take a look to all the UDFs and their signatures in the [functions
 ## Retrieving UASTs with the UDF `uast`
 
 ```sql
-SELECT file_path, uast(blob_content, language(file_path)) FROM files;
+SELECT file_path,
+       uast(blob_content, language(file_path))
```

Comment:
we talked about having better examples with UAST, because this is not giving any useful information to the user. Could we change this to get some specific tokens from HEAD commits, per example (because this query is really heavy too)?